src/styles/blocks/content.scss

```diff
@@ -0,0 +1,23 @@
+.content {
+  padding: 50px 40px 0;
+  margin: 0;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  gap: 46px 48px;
+
+  /* The gap is needed because when change grid you reset gap values */
+  @include screen-mixin-grid-level-1 {
```

Comment:
I like your use of mixins, but I'd like to suggest a rename for them to allow them to be used more often, which is the purpose of a mixin (imagining this is part of a bigger project).

So, the way I see it, **screen-mixin-grid-level-1** is used for **small-screens** or **mobile**, **screen-mixin-grid-level-2** is used for *medium-screens** or **tablets** and **screen-mixin-grid-level-3** is used for **large-screens** or **desktop**.

Giving them a name as mentioned makes it possible to re-use them in the future for other purposes other than only the column count, imagine that you wanted to change the background color of your page based on screen size too, in this way, the name would remain meaningful for all usage.